src/components/datepicker/OcDatepicker.vue

```diff
@@ -0,0 +1,52 @@
+<template>
+  <date-picker class="oc-datepicker" v-bind="$attrs" v-on="$listeners">
+    <template #default="{ inputValue, togglePopover, hidePopover }">
+      <!-- @slot Default slot to use as the popover anchor for datepicker -->
+      <slot :inputValue="inputValue" :togglePopover="togglePopover" :hidePopover="hidePopover" />
+    </template>
+  </date-picker>
+</template>
+
+<script>
+import DatePicker from "v-calendar/lib/components/date-picker.umd"
+
+/**
+ * Datepicker component based on [v-calendar](https://vcalendar.io/)
+ */
+export default {
+  name: "OcDatepicker",
+  status: "ready",
+  release: "1.0.0",
+
+  components: { DatePicker },
+
+  inheritAttrs: true,
+}
+</script>
+
+<style lang="scss">
+.oc-datepicker .vc-highlight {
```

Comment:
Probably needs some sort of background color styling via CSS variables, too :)

Reply:
Not sure if I get you. Do you mean creating a new variable for this background?

Reply:
Let me check how theming affects this component before I get back to you on this!

Reply:
The state like it is now already helps for unblocking development in web, right? Let's merge this, make an rc-release to unblock work in `web`, and do the theming support in a separate ODS PR, ok?

Reply:
Sure, could be (would also help removing UiKit 😬 )